src/vivarium_public_health/treatment/magic_wand.py

```diff
@@ -62,9 +58,10 @@ def setup(self, builder: Builder) -> None:
 
     def intervention_effect(self, index, value):
         if self.config["target_value"] != "baseline":
-            pop = self.population_view.get(index)
-            affected_group = pop[
-                pop.age.between(self.config["age_start"], self.config["age_end"])
-            ]
-            value.loc[affected_group.index] = float(self.config["target_value"])
+            affected_group_idx = self.population_view.get_population_index(
+                index,
+                query_columns="age",
```

Comment:
Get rid of query_columns, and if there's no trailing comma, I think this will fit on one line.